DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h

```diff
@@ -33,18 +36,21 @@ struct BCMask {
 struct CTPInput {
   CTPInput() = default;
   std::string name;
-  o2::detectors::DetID::ID detID;
+  std::string level;
+  std::string detName;
```

Comment:
note that the DetID has a method getName, i.e. instead of storing the name you can simply call
`DetID::getName(detID)``.
Alternatively, you can use DetID directly as a data member, instead of DetID::ID.

Reply:
added full DetID to CTPInput